iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/DefaultNamingStrategy.java

```diff
@@ -0,0 +1,105 @@
+/**
+ * Copyright (c) 2023, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+package com.powsybl.iidm.modification.topology;
+
+import com.powsybl.iidm.network.*;
+
+import java.util.List;
+
+/**
+ * Default naming strategy used if no other naming strategy is specified.
+ * @author Nicolas Rol <nicolas.rol@rte-france.com>
+ */
+public class DefaultNamingStrategy implements NamingStrategy {
+
+    private static final String SEPARATOR = "_";
+    private static final String DISCONNECTOR_NAMEBASE = "DISCONNECTOR";
+    private static final String BREAKER_NAMEBASE = "BREAKER";
+    private static final String SWITCH_NAMEBASE = "SW";
+    private static final String BUS_NAMEBASE = "BUS";
+
+    @Override
+    public final String getName() {
+        return "Default";
+    }
+
+    @Override
```

Comment:
I think the prefix should be retrieved directly in the methods and not defined in the interface. Basically these methods should be private

Reply:
If I understand correctly, you mean that those two methods (`getSectioningPrefix` and `getChunkPrefix`) are specific to the default naming strategy and will not likely be implemented in another naming strategy. 

You suggest that the sectioning prefix and the chunk prefix should be calculated directly in the `getDisconnectorBetweenChunksId`, `getBreakerId`, `getDisconnectorBetweenChunksId` and `getDisconnectorId` methods.

I don't know what the usual naming strategies are but I think this would add many parameters to the aforementioned methods, thus making them less generic in my opinion.